support-frontend/assets/pages/[countryGroupId]/components/checkoutLayout.tsx

```diff
@@ -25,6 +25,7 @@ const secureTransactionIndicator = css`
 `;
 
 const darkBackgroundContainerMobile = css`
+	display: flex;
```

Comment:
I've applied it to specific components instead of the last child of the page scaffold (which has `flex: 1` set on it [here](https://github.com/guardian/support-frontend/blob/35caef3ac84963dd783d81c5c41e955d9362a32a/support-frontend/assets/components/page/pageScaffold.tsx#L24)), because the thank you page uses the same scaffold but changing it to a flexbox changes the layout completely

Reply:
Ah ok, presumably the `flex` affects the thankyou component arrangement.👍

Reply:
Yes, maybe we could refactor the thank you page to make it adapt to being inside a flexbox? Wasn't sure it made sense. What do you think?

Reply:
Not sure its worth it, there are only 2 flex lines needed and to adapt the thankyou would probably be more lines anyway